pkg/endpoint/endpoint.go

```diff
@@ -2401,3 +2403,9 @@ func (e *Endpoint) IPs() []net.IP {
 	}
 	return ips
 }
+
+// InsertEvent is called when the endpoint is inserted into the endpoint
+// manager. The endpoint must be read locked.
+func (e *Endpoint) InsertEvent() {
+	e.getLogger().Info("New endpoint")
```

Comment:
These seem a little lacking in detail. Managing endpoint/stopped managing endpoint might be better but that implies that the endpoint might continue to exist (Which isn't normally the case but is correct, since we don't release the IP until the runtime deletes the container, right?)

Reply:
The new and remove event directly correlate with the use of the IP. It has been allocated just before new event and it has been released just before remove event.